html/user/prefs_edit.php

```diff
@@ -78,6 +77,7 @@
             } else {
                 $main_prefs = $prefs;
                 prefs_privacy_parse_form($user);
+                prefs_consent_parse_update($user);
             }
 
             project_prefs_update($user, $main_prefs);
```

Comment:
I'm not certain what the source of the issue is, but I created a new account via the client and went to enable my stats export through the project preferences.  When I submitted it I got the message:

```
Notice: Undefined index: default_venue in /root/project/html/inc/prefs.inc on line 689

Warning: Cannot modify header information - headers already sent by (output started at /root/project/html/inc/prefs.inc:689) in /root/project/html/user/prefs_edit.php on line 89
Back to preferences 
```

Reply:
I'm blocked in further testing due to this issue.  Let me know when fixed and I will resume testing.

Reply:
Quick question:
>  I created a new account via the client and went to enable my stats export through the project preferences.

I assume you mean you went to the project's Web site, went to the project preferences page, and attempted to save/submit the changes? 

I will try to debug this later today and get back to you.

Reply:
Yes.  Complete path.  Create account either via client or via website.  Go to project preferences and edit preferences on default profile.  Attempt to save settings.  Get error.  Thanks.

Reply:
Fixed in https://github.com/BOINC/boinc/pull/2504/commits/683c5629bc566c2ee79ba6b0f79a6399155b799e, I removed a necessary line accidentally in a development mistake. Reverting the line should solve the bug.